app/views/layouts/rad_mailer.html.haml

```diff
@@ -60,7 +60,7 @@
                 .footer-item
                   Click
                   = link_to 'here', "#{protocol}://#{RadConfig.host_name!}/rad_common/notification_settings"
-                  to opt out or change your notification settings.
```

Comment:
I worried that users might think this opts them out upon clicking, but it doesn't, it just takes them to the notification settings page